process.php

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>
+</head>
+<body>
+
+<?php
+    $weight = $_POST['weight'];
+    $height = $_POST['height'];
+
+    function bmi($weight, $height)
+	{return ($weight / ($height * $height)) * 703;}
+	$bmi_value = bmi($weight, $height);
+	
+    if ($bmi_value < 18) {$label = "underweight";}
+	else if ($bmi_value > 30) {$label ="overweight";}
+	else {$label ="normal";}

```

Comment:
solid if/else structure! I like that you leave the middle scenario until your else - that is most efficient 😎